### ISSUE TITLE
Cannot input the first character with Chinese IME

### ISSUE THREAD

**Issue by Pavek** [quilljs/quill#626]
This is quite specific test-case reproduced on different Windows machines with Chinese Traditional keyboard input. Unfortunately, this is quite nasty.

While typing the very first character in editor with the input method specified below, the caret goes back to start position every time you press a key, essentially making it impossible to start typing. **Important note**: after typing a space character (' ') you can then type normally!

Language: Chinese (Traditional, Taiwan)
Keyboard: Chinese (Traditional) - New Phonetic
Preferred keyboard layout - HanYu Pinyin

Probably, it's an edge case for issue #213.

**Steps for Reproduction**
1. Visit quilljs.com (yes, reproduced on demo page!)
2. Remove all content from the editor (make it empty)
3. Switch to Chinese Keyboard specified above
4. Start typing any chars

**Expected behavior**: The mini-popup appears underneath the typed char and one can complete the hieroglyph.

**Actual behavior**: The caret moves back to initial position and mini-popup does not appear, essentially making it impossible to type. Pressing space solves the issue.

**Platforms**: IE11 on Win7 (_not reproduced_ on latest FF and Chrome)

**Version**: v0.20.1

**Comment by jhchen** [comment]
Thanks for the report. I'll see if I can reproduce it in the 1.0 development branch and look into a fix if so.

**Comment by Pavek** [comment]
Here's the crazy workaround: https://jsbin.com/fidivelale/1/edit?html,js,output
Notes:
- We use react-quill and this jsbin contains stripped-down version of our implementation
- Notice the `&nbsp;` on the line 3 -- this is the key. Without it one cannot type Chinese characters on IE11.

**Comment by KennyWho** [comment]
@jhchen I have same problem like this, it can reproduction everytime, but it's on chrome.

**Reproduction**
1. Visit http://codepen.io/quill/pen/KzZqZx the official example.
2. Same as above.

Platforms: Chrome v-52.0.2743.116 m (64-bit) on Win10. I test on OS X, it has the same problem.

**Comment by yachtingchen** [comment]
Here is how the problem looks like:
expected behavior: (recording from Chrome)

![expected3](https://cloud.githubusercontent.com/assets/8899072/23338196/223d4b18-fc3f-11e6-9d57-fdfc6232e874.gif)

actual behavior: (recording from IE11)

![actual3](https://cloud.githubusercontent.com/assets/8899072/23338200/30c22e9c-fc3f-11e6-9862-8049949e092f.gif)

Environment: Windows 8
Input method: Chinese (Traditional, Taiwan) Microsoft Bopomofo

**Comment by yeanzhi** [comment]
The 56 version of the chrome can reproduce this bug

**Comment by KennyWho** [comment]
plz disable your chrome extend，and have a try.

**Comment by yeanzhi** [comment]
@KennyWho ， when I disable evernote extend ,everything is all right. But why? I found ScrollBlot's MutationObserver (in parchment)  give a wrong mutations  when I able evernote extend . 
【Right】
![3 -06-2017 15-24-26](https://cloud.githubusercontent.com/assets/4659887/23600357/43267910-0281-11e7-8dfe-f140373cb309.gif)
【Wrong】
![3 -06-2017 15-24-37](https://cloud.githubusercontent.com/assets/4659887/23600360/47a97960-0281-11e7-8d09-a7828a1073e7.gif)

But ,when I write a  contenteditable div like this
![image](https://cloud.githubusercontent.com/assets/4659887/23600255/a16f9a5c-0280-11e7-91ec-0eb3c34e4c56.png)
No matter whether I disable the extend, everything is fine.

**Comment by ehe888** [comment]
After some digging and testing. I found at least in Chrome this problem is caused by Evernote Extension. I have tried safari without any issue. 
But in Chrome if I enable Evernote extension the issue reproduced. I have some other Chrome extensions such like grammarly which will live check my typing but only Evernote leads to this problem. I don't know what Evernote extension do to the input but it apparently intercepts the input content and not properly passes it down to next layer.  

But when I tested in IE Edge another wired problem happened. When I input Chinese at the beginning of an empty line, the first character I typed will be directly treated as English character and placed into the cursor position. Thie behavior breaks my normal Chinese Pinyin input, change my consecutive input into 1 leading English character and some Pinyin characters following.

See the attached screen shot. I am typing shurufa which is Pinyin of Chinese "输入法", but I got "s + hurufa".

![image](https://cloud.githubusercontent.com/assets/706864/24076946/f753db4e-0bfa-11e7-9429-311ba35ba09b.png)

Now, it seems the issue only happened in IE. I will keep digging on this issue.
